jobbergate-cli/jobbergate_cli/schemas.py

```diff
@@ -127,7 +127,7 @@ class ApplicationResponse(pydantic.BaseModel, extra=pydantic.Extra.ignore):
     Describes the format of data for applications retrieved from the Jobbergate API endpoints.
     """
 
-    id: int
+    application_id: int = pydantic.Field(alias="id")
```

Comment:
I wonder if we could apply an alias when exporting the model, so the code can stay as it is, but the names presented to the users are different.

Just thinking aloud, because the implementation on this PR is good to go as well.